README.md

```diff
@@ -60,11 +60,13 @@ This means we follow the same installation than above, but using the
 ```sh
 λ opam switch 4.02.3+buckle-master
 λ eval `opam config env`
-λ opam install odoc
+λ opam pin add odoc.dev git+https://github.com/ocaml/odoc
 λ odoc --version
 1.2.0
```

Comment:
The output for `--version` will be different for the pinned installation. Currently it's `v1.1.1-743-gb6e4edc`. You can either remove this line entirely or put the current one as an example.

Reply:
It's better to remove this line because after the next push to develop branch it could be different.

Reply:
To be honest that kind of information is becoming so outdated so quickly I'm not sure it's the right place to put this.

Reply:
I was thinking about it. The first change was to stay with `opam install odoc` and put information below the instruction.

Reply:
@dbuenzli are you referring to the installation by pinning or the version?

In any case, many people seem eager to try odoc with BuckleScript, this instructions will be helpful for them. I'll make sure we update the README once the next version of odoc is released.

Reply:
Pinning switch creation that includes a particular version, the toy shell script to generate the docs. etc. etc. I think this information should rather go to somewhere were bucklescript users refer to when they need documentation about their platform.

Reply:
Conceptually I agree, it's entirely out of odoc's scope to worry about BuckleScript's integration. In practice though, odoc is being only experimentally used by some packages (like [bs-react-native](https://github.com/reasonml-community/bs-react-native#readme)). It's currently not the officially recommended documentation generator for BuckleScript (it still uses ocamldoc AFAIK).

When things get more stable I would expect to have a proper external integration with BuckleScript's build system on the same level as we have with dune. For now I personally don't see any risks in _provisionally_ helping with the integration as long as it stays out of odoc's core.

Submitting a PR to BuckleScript's website with instructions to use odoc is another option. I don't know if @ostera and @aantron considered this when the BuckleScript section/script were added.

In any case, I will merge this PR as it's both harmless and useful. Further decisions can be discussed separately.